src/foundry/prosemirror/schema/image-node.d.mts

```diff
@@ -0,0 +1,19 @@
+import { MarkSpec, Node, NodeSpec } from "prosemirror-model";
+import type SchemaDefinition from "./schema-definition.d.mts";
+
+export default ImageNode;
+
+/**
+ * A class responsible for encapsulating logic around image nodes in the ProseMirror schema.
+ */
+declare class ImageNode extends SchemaDefinition {
+  static override tag: "img[src]";
+
+  static override get attrs(): Record<string, any>;
```

Comment:
We use `Record<string, unknown>` over `Record<string, any>` because it adds a bit of safety. For example if you write `ImageNode.attrs.detials.name` and then try to use it it'll work fine despite the fact that in this example `detials` is a typo of `details`.